internal/config/client_configuration.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+type ClientConfiguration struct {
+	Groundstation struct {
+		Hostname string `toml:"hostname"`
+		Port     int    `toml:"port"`
+	} `toml:"groundstation"`
+	Satellite struct {
```

Comment:
NIT: These should probably be their own types, so that we don't need the tag data everywhere.

Reply:
I'd rather keep code locality as much as possible on this. This was an explicit decision not to extract out types as they're not used elsewhere. I would start to get worried if the configuration bundle gets massive, but by then it belongs in it's own file.

Reply:
But they are used twice (in definition, and in default func). I'd like to be able to pass around just one of them at some point. It also makes the default funcs quite unreadable

Reply:
That's fair. We can look into that then.